examples/webhook.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from os import getenv
+
+from aiohttp.web import run_app
+from aiohttp.web_app import Application
+
+from aiogram.webhook.aiohttp_server import SimpleRequestHandler, setup_application
+from aiogram import Dispatcher, Router, Bot, types, F
+from aiogram.methods import SendMessage
+
+
+TELEGRAM_TOKEN = getenv("TELEGRAM_TOKEN")
+
+# Webhook settings
+WEBHOOK_HOST = getenv("WEBHOOK_HOST", "https://your.domain")
+WEBHOOK_PATH = getenv("WEBHOOK_PATH", "/path/to/api")
+WEBHOOK_URL = f"{WEBHOOK_HOST}{WEBHOOK_PATH}"
+
+# Webserver settings
+WEBAPP_HOST = getenv("WEBAPP_HOST", "localhost")  # or ip
+WEBAPP_PORT = getenv("WEBAPP_PORT", 3001)
+
+
+router = Router()
+
+
+@router.message(F.text)
+async def echo(message: types.Message):
+    # Regular request, add bot: Bot to handler kwargs
+    # await bot.send_message(message.chat.id, message.text)
+
+    # or reply INTO webhook
+    return SendMessage(chat_id=message.chat.id, text=message.text)
+
+
+@router.startup()
```

Comment:
I'm not sure if it's good to use the same router for the TelegramEventObserver and EventObserver. But this is not so important for this example.

Reply:
I'ts ok.